resources/src/tldrPageUrl.tsx

```diff
@@ -0,0 +1,23 @@
+enum FileAction {
+  VIEW,
+  CREATE,
+}
+
+function tldrPageUrl(action: FileAction, os: string, page: string, language: string) {
+  const languageSuffix = language === 'en' ? '' : '.' + language
+
+  const baseUrl = 'https://github.com/tldr-pages/tldr'
+  const filePath = `/main/pages${languageSuffix}/${os}/${page}.md`
+
+  if (action === FileAction.CREATE) {
+    return baseUrl + '/new' + filePath + `?filename=${page}.md`
+  }
+
+  if (action === FileAction.VIEW) {
+    return baseUrl + '/blob' + filePath
+  }
+
+  throw new Error('Unknown GitHubFileAction: ' + action)
```

Comment:
Not sure about this, but the type of `action` is `FileAction`—
```suggestion
  throw new Error('Unknown FileAction: ' + action)
```

Reply:
Correct, Some time ago I renamed the type to `FileAction`, so we should also change the error message 👍